### PR TITLE
[create:comments] Add new command.

### PR DESCRIPTION
**Pull request by gnuget**
**Problem/Motivation**
I've been working lately in some issues related with the comment module, and this command is handy  for me.

### CODE REVIEW

File: src/Utils/Create/Comments.php [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        $nid,
+        $limit,
+        $titleWords,
+        $timeRange
+    ) {
+        $comments = [];
+        $node = Node::load($nid);
+
+        for ($i=0; $i<$limit; $i++) {
+            $comment = Comment::create([
```
Comment (gnuget) [inline src/Utils/Create/Comments.php]: We need to rewrite this if you guys decide to no to use the static methods.

File: src/Utils/Create/Comments.php [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+use Drupal\comment\Entity\Comment;
+
+/**
+ * Class Nodes
+ * @package Drupal\Console\Utils
+ */
+class Comments extends Base
+{
+    /* @var array */
+    protected $bundles = [];
```
Comment (gnuget) [inline src/Utils/Create/Comments.php]: I don't use this variable 😅 I will send another commit and remove it.

File: src/Utils/Create/Comments.php [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+    /**
+     * Comments constructor.
+     *
+     * @param EntityManagerInterface $entityManager
```
Comment (gnuget) [inline src/Utils/Create/Comments.php]: If you decided to apply my other PR, we need to update this and use the entityTypeManager,  I can do it once you decide of the other PR will be merged or not.

File: src/Utils/Create/Comments.php [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+     */
+    public function __construct(
+        EntityManagerInterface $entityManager,
+        DateFormatterInterface $dateFormatter
+    ) {
+        parent::__construct($entityManager, $dateFormatter);
+    }
+
+    /**
+     * @param $nid,
```
Comment (gnuget) [inline src/Utils/Create/Comments.php]: This  variable must be named cid 😐

File: src/Utils/Create/Comments.php (hunk 3 above)
Comment (jmolivas) [inline src/Utils/Create/Comments.php]: I am about to merge the #2014

File: src/Utils/Create/Comments.php (hunk 1 above)
Comment (jmolivas) [inline src/Utils/Create/Comments.php]: Please use the injected object.

Comment (gnuget) [inline src/Utils/Create/Comments.php]: Sure, I will do it asap.  thx

File: src/Utils/Create/Comments.php (hunk 3 above)
Comment (gnuget) [inline src/Utils/Create/Comments.php]: OK,  I will send another commit to fix this.

**Pull request merged**